jenkins_jobs/trigger_weekly_host_os_build/pre_build_script.sh

```diff
@@ -23,30 +23,34 @@ write_comment() {
 	   "$pr_number" "$comment_text"
 }
 
-# Upgrade versions
-# sudo yum install rpmdevtools
-python host_os.py \
-       --verbose \
-       upgrade-versions \
-           --updater-name "$GITHUB_BOT_NAME" \
-           --updater-email "$GITHUB_BOT_EMAIL" \
-           --push-repo-url "ssh://git@github.com/${GITHUB_BOT_USER_NAME}/versions.git" \
-           --push-repo-branch "$COMMIT_BRANCH"
+upgrade_versions() {
+    python host_os.py \
+           --verbose \
+           upgrade-versions \
+               --updater-name "$GITHUB_BOT_NAME" \
+               --updater-email "$GITHUB_BOT_EMAIL" \
+               --push-repo-url "ssh://git@github.com/${GITHUB_BOT_USER_NAME}/versions.git" \
+               --push-repo-branch "$COMMIT_BRANCH"
+}
+
+create_release_notes() {
+    python host_os.py \
+           --verbose \
+           release-notes \
+               --build-versions-repository-url "ssh://git@github.com/${GITHUB_BOT_USER_NAME}/versions.git" \
+	           --build-version "$COMMIT_BRANCH" \
+	           --updater-name "$GITHUB_BOT_NAME" \
+	           --updater-email "$GITHUB_BOT_EMAIL" \
+	           --push-repo-url "ssh://git@github.com/${GITHUB_BOT_USER_NAME}/${GITHUB_ORGANIZATION_NAME}.github.io.git" \
+	           --push-repo-branch "$COMMIT_BRANCH"
```

Comment:
Those lines have tabs, they are not aligned with `--build-versions-repository-url`